elements/bulbs-video/components/revealed.js

```diff
@@ -274,6 +303,16 @@ export default class Revealed extends React.Component {
     }
   }
 
+  handlePauseEvent (reason) {
+    if (reason.pauseReason === 'external') {
+      return;
+    }
+    else if (reason.pauseReason === 'interaction') {
```

Comment:
we can assume its an ad if there is no `pauseReason`, in fact `adPause` is its own event.  But we won't be able to determine if scrolling the player out of view paused the ad or if the user paused it.